### PR TITLE
Bugfix for autosizing when detailed performance capacity fractions are provided

### PR DESCRIPTION
**Pull request by yzhou601**
## Pull Request Description

The autosizing to convert design condition load to rated condition capacity (47F) is based on the 47F maximum capacity fraction, which is not correct. It should apply the nominal capacity fraction (which is 1.0).
cc @jonwinkler 

## Checklist

Not all may apply:

- [ ] Schematron validator (`EPvalidator.xml`) has been updated
- [ ] Sample files have been added/updated (`openstudio tasks.rb update_hpxmls`)
- [x] Tests have been added/updated (e.g., `HPXMLtoOpenStudio/tests/test*.rb` and/or `workflow/tests/test*.rb`)
- [ ] Documentation has been updated
- [x] Changelog has been updated
- [x] `openstudio tasks.rb update_measures` has been run
- [x] No unexpected changes to simulation results of sample files

### CODE REVIEW

**Comment by yzhou601** [comment]
| case8
-- | --
design temperatgure | 6.62
outdoor temp | capacity fraction
67 | 1.641
47 | 1.216
17 | 0.913
5 | 0.867
  |  
slope | 0.003833333
fraction design to nominal | 0.873133333
fraction design to max 47F | 0.718037281
design load | 55527.13901
rated capacity EXPECTED | 63595.25732
rated capacity ACTUAL | 77331.8329

File: HPXMLtoOpenStudio/tests/test_hvac_sizing.rb [hunk 1]
```diff
@@ -1345,6 +1346,35 @@ def test_autosizing_limits
...
+    hpxml, hpxml_bldg = _create_hpxml('base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml')
+    hpxml_bldg.header.heat_pump_sizing_methodology = HPXML::HeatPumpSizingMaxLoad
+    hpxml_bldg.heat_pumps[0].backup_heating_capacity = nil
+    hpxml_bldg.heat_pumps[0].heating_capacity = nil
+    hpxml_bldg.heat_pumps[0].cooling_capacity = nil
+    hpxml_bldg.heat_pumps[0].heating_detailed_performance_data.find { |dp| dp.outdoor_temperature == HVAC::AirSourceHeatRatedODB && dp.capacity_description == HPXML::CapacityDescriptionMaximum }.capacity_fraction_of_nominal = 1.2
+    hpxml_bldg.heat_pumps[0].cooling_detailed_performance_data.find { |dp| dp.outdoor_temperature == HVAC::AirSourceCoolRatedODB && dp.capacity_description == HPXML::CapacityDescriptionMaximum }.capacity_fraction_of_nominal = 1.1
+    XMLHelper.write_file(hpxml.to_doc, @tmp_hpxml_path)
+    _model, _hpxml, hpxml_bldg = _test_measure(args_hash)
+    assert_equal(hpxml_bldg.heat_pumps[0].heating_capacity, htg_cap_orig)
```
Comment (yzhou601) [inline HPXMLtoOpenStudio/tests/test_hvac_sizing.rb:1373]: The unit test here verifies that the autosized rated capacity is the same as original, regardless of maximum speed capacity fractions of nominal at rated temperatures. The fractions will be applied later when pop up the capacities in detailed performance data points.

**Comment by shorowit** [comment]
@yzhou601 I see that there are no CI diffs, but I was thinking that `base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml` might show some diffs. Why does that file not change?

**Comment by yzhou601** [comment]
> @yzhou601 I see that there are no CI diffs, but I was thinking that `base-hvac-air-to-air-heat-pump-var-speed-detailed-performance-autosize.xml` might show some diffs. Why does that file not change?

This issue only shows up when the maximum speed capacity fraction at rated temperature is over 1.0. For that test file we configured it to be 1.0 at maximum, 47F. Besides, the test file uses HERS sizing methodology, [where the loads override the sized capacity](https://github.com/NREL/OpenStudio-HPXML/blob/master/HPXMLtoOpenStudio/resources/hvac_sizing.rb#L2474-L2487). See my unit test, it reveals the issue and gets fixed with the changes in this PR.

**Review by shorowit — APPROVED** [review]

**Pull request merged**